src/stories/Form.stories.js

```diff
@@ -50,6 +51,38 @@ const Template = (args) => {
     );
 };
 
+/**
+ * Story to exhibit the native event handlers for TextInput in the Form Component
+ * @param {Object} args
+ * @returns {JSX}
+ */
+const WithNativeEventHandler = (args) => {
+    const [log, setLog] = useState('');
+
+    // Form consumes data from Onyx, so we initialize Onyx with the necessary data here
+    FormActions.setIsSubmitting(args.formID, args.formState.isSubmitting);
+    FormActions.setServerErrorMessage(args.formID, args.formState.serverErrorMessage);
+    FormActions.setDraftValues(args.formID, args.draftValues);
+
+    return (
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        <Form {...args}>
+            <TextInput
+                label="Routing number"
+                inputID="routingNumber"
+                onChangeText={setLog}
+                isFormInput
+                shouldSaveDraft
+            />
+            <Text>
+                Entered routing number:
+                {' '}
+                {log}
```

Comment:
use template literals here?

Reply:
It can be changed to 
```js
   Entered routing number:
                {` ${log}`}
```
So I don't think it is worth it.

Reply:
Oh I meant this - {\`Entered routing number: ${log}\`}

Reply:
Done